packages/core/strapi/lib/commands/index.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const { Command } = require('commander');
+
+const availableCommands = [
```

Comment:
I'm not sure I'm convinced with this approach of harcoding a list to make dynamic requires when it would be simple to just require them instead here. I'm wondering if we aren't also lossing some loading time with this 🤔

Reply:
Do you mean you'd prefer to load them automatically from what is found in the actions dir?

Reply:
Shouldn't we make this dynamic to allow plugins to inject new commands too?

Reply:
Considering we are just hardcoding the list here I don't see why we make the requires dynamic. We just make the startup slower but don't gain flexibility.

Either I would just require them or move to using readdir/glob to detect the existing commands if we want it to be 100% dynamic (not sure why that would want that to tbh considering we will want plugin & user commands to be dynamic not the default ones anyway)

Reply:
It was mostly a tradeoff to enforce the path structure, but static is fine too, just updated it.

@derrickmehaffy This is a first step towards allowing plugins to inject commands. I didn't want to start on it until this was cleaner though :)

Next steps will be something like:
- moving the detection and loading of the `strapi` object from the individual commands to a higher level like in `buildStrapiCommand` (and the command builders can use a commander hook when they require it)
- at that point, since`strapi` will be available in the builder, we can check it for plugin and user commands which would be added by plugins during the load process